docs/_extra/css/extra.css

```diff
@@ -174,6 +174,14 @@
   .md-typeset table:not([class]) {
   font-size: 0.75rem
   }
+
```

Comment:
This shouldn't be needed. When you hide the TOC, the table should automatically expand, as it has for example here: https://docs.n8n.io/code/builtin/output-other-nodes/

Did you find it wasn't working without the CSS?

Reply:
I noticed that the table width in the `enviornment-variables/database` page would be inconsistent without this css, but maybe I'm missing something here.

Reply:
Agreed: leave it in. Add a comment to say what you've done.

Reply:
Reference: [Force data table to use full width #3530](https://github.com/squidfunk/mkdocs-material/discussions/3530) from discussions in mkdocs-material 
Added a comment in the css file.